src/components/Chart.tsx

```diff
@@ -0,0 +1,156 @@
+/** Load dependencies */
+import React from "react";
+import { Chart as ChartJS } from "chart.js";
+import "chartjs-plugin-datalabels";
+
+/** Load types */
+import { Metric } from "../types";
+
+/** Types */
+type Context =
+  | string
+  | CanvasRenderingContext2D
+  | HTMLCanvasElement
+  | ArrayLike<CanvasRenderingContext2D | HTMLCanvasElement>;
+
+type RefContext =
+  | string
+  | ((instance: HTMLCanvasElement | null) => void)
+  | React.RefObject<HTMLCanvasElement>
+  | null
+  | undefined;
```

Comment:
Those `Context` and `RefContext` Types are too generic (too many possibilities), can't we me more especific/precise?